DependencyInjection/FOSUserExtension.php

```diff
@@ -108,6 +108,7 @@ public function load(array $configs, ContainerBuilder $container)
                 'user_class' => 'fos_user.model.user.class',
             ),
         ));
+        $container->setParameter('fos_user.from_email', array($config['from_email']['address'] => $config['from_email']['sender_name']));
```

Comment:
please don't create a new `fos_user.from_email`. Instead, set dummy values for these emails in case the other features are not used (they won't be used anyway in such case)

Reply:
@stof fixed i've put dummy values to `mailer.xml`